### PR TITLE
[DOCFIX] Update the Chinese version of Contributing-to-Alluxio.md

### PR DESCRIPTION
**Pull request by saltylin**

### CODE REVIEW

**Comment by saltylin** [comment]
Jenkins, test this please

File: docs/cn/Contributing-to-Alluxio.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ group: Resources
 
 感谢您对alluxio的兴趣！我们非常感谢任何的新特性或者修复的贡献。
```
Comment (RongGu) [inline docs/cn/Contributing-to-Alluxio.md]: 'alluxio' -> 'Alluxio'

File: docs/cn/Contributing-to-Alluxio.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ group: Resources
...
 
+## 新的贡献者
```
Comment (RongGu) [inline docs/cn/Contributing-to-Alluxio.md]: remove ‘的’

File: docs/cn/Contributing-to-Alluxio.md [hunk 3]
```diff
@@ -10,6 +10,10 @@ group: Resources
...
+
+如果你是一个新的开源贡献者，请先浏览[开始贡献源码向导](Contributing-Getting-Started.html)以熟悉如何向Alluxio贡献源码。
```
Comment (RongGu) [inline docs/cn/Contributing-to-Alluxio.md]: remove ‘的’

File: docs/cn/Contributing-to-Alluxio.md [hunk 4]
```diff
@@ -10,6 +10,10 @@ group: Resources
...
+
 ### Alluxio初始任务
 
 新的开发者可以先进行以下列出的任务，从而能够更加熟悉Alluxio：
```
Comment (RongGu) [inline docs/cn/Contributing-to-Alluxio.md]: remove ‘的’

**Comment by RongGu** [comment]
Thanks for the contributation. I have some minor comments. Please handle that, thanks! @saltylin

File: docs/cn/Contributing-to-Alluxio.md (hunk 1 above)
Comment (saltylin) [inline docs/cn/Contributing-to-Alluxio.md]: Updated

File: docs/cn/Contributing-to-Alluxio.md (hunk 2 above)
Comment (saltylin) [inline docs/cn/Contributing-to-Alluxio.md]: Updated

File: docs/cn/Contributing-to-Alluxio.md (hunk 3 above)
Comment (saltylin) [inline docs/cn/Contributing-to-Alluxio.md]: Updated

File: docs/cn/Contributing-to-Alluxio.md (hunk 4 above)
Comment (saltylin) [inline docs/cn/Contributing-to-Alluxio.md]: Updated

**Comment by saltylin** [comment]
Thanks very much for your careful review!! I have addressed your comments! @RongGu

**Comment by RongGu** [comment]
LGTM.

**Comment by ifcharming** [comment]
LGTM

**Comment by ifcharming** [comment]
alluxio-bot, check this please

**Pull request merged**